discovery/eureka_test.go

```diff
@@ -53,7 +53,14 @@ func TestMain(m *testing.M) {
 func TestEurekaClientNoEureka(t *testing.T) {
 	_, err := NewEurekaClient("http://localhost:9999/thisshouldntwork")
 	if err != errNoEurekaConnection {
-		t.Fatal("We shouldnt reach eureka if Eureka hostname/port is completely wrong")
+		t.Fatal("We shouldnt reach eureka if Eureka hostname/port is completely wrong. Actual err:", err)
+	}
+}
+
+func TestEurekaClientEurekaDoesNotReply(t *testing.T) {
+	_, err := NewEurekaClient("http://192.0.2.1:9999/thisshouldtimeout")
```

Comment:
"Addresses starting with "192.0.2.", "198.51.100.", or "203.0.113." are reserved for use in documentation and sample configurations.  They     should never be used in a live network configuration.  No one has permission to use these addresses on the Internet." 
I guess doing testing is no documentation...

Reply:
Can you suggest another IP range which traffic will be dropped for sure? In fact, these networks are referenced as "TEST NETWORKs" in the RFC, so that was the best option I found...

Reply:
you were right, don't change it.